app/calculations/tweets_count_per_hashtag.rb

```diff
@@ -2,19 +2,27 @@ class TweetsCountPerHashtag < Patterns::Calculation
   private
 
   def result
-    tweets_count.sort do |current_hashtag, next_hashtag|
-      comparison = next_hashtag[1] <=> current_hashtag[1]
-      comparison.zero? ? current_hashtag[0] <=> next_hashtag[0] : comparison
-    end
+    sorted_hashtag_stats.take(limit)
   end
 
   def organization
     options.fetch(:organization)
   end
 
+  def limit
+    options.fetch(:limit)
+  end
+
   def tweets_count
     HashTag.in_organization(organization).inject([]) do |count, tag|
-      count << [tag.name, tag.tweets.count]
+      count << [tag.name, tag.tweets_count]
```

Comment:
rename `count` to `stat` and pass there object: `{name: tag.name, count: tag.tweets_count}` - it'll be better for understanding that in the front-end code, so that instead of using cryptic indexes: `[0]` or `[1]`, we'll be able to call e.g.: `stats.name`